hashes/src/ripemd160.rs

```diff
@@ -465,12 +467,24 @@ mod tests {
             },
         ];
 
-        for test in tests {
+        for mut test in tests {
             // Hash through high-level API, check hex encoding/decoding
             let hash = ripemd160::Hash::hash(test.input.as_bytes());
             assert_eq!(hash, test.output_str.parse::<ripemd160::Hash>().expect("parse hex"));
             assert_eq!(&hash[..], &test.output[..]);
             assert_eq!(&hash.to_string(), &test.output_str);
+            assert_eq!(
+                ripemd160::Hash::from_bytes_ref(
+                    <&[u8; 20]>::try_from(&*test.output).expect("known length")
+                ),
+                &hash
+            );
+            assert_eq!(
+                ripemd160::Hash::from_bytes_mut(
+                    <&mut [u8; 20]>::try_from(&mut *test.output).expect("known length")
+                ),
+                &hash
+            );
```

Comment:
Wouldn't it be better to just change `Vec<u8>` to `[u8; 20]` above to avoid this noise of converting?

Also, ideally these tests should run through `miri` but we might need to `cfg(not(miri))` other some things.